Re-Introduces Formatting for Date Fields across Sheets

colinfrederickson opened the pull request:
- Formats date fields to YYYY-MM-DD across all sheets.

this PR is necessary based on the new formatting behavior of the date field type.

@johnmosesman - was unable to fully test locally using the Flatfile application. if possible, can we set up an environment to test this end to end locally?

colinfrederickson committed:
Re-Introduces Formatting for Date Fields across Sheets

- Formats date fields to YYYY-MM-DD across all sheets.

colinfrederickson committed:
Re-Introduces Formatting for Date Fields across Sheets

- Formats date fields to YYYY-MM-DD across all sheets.

colinfrederickson committed:
Updates Logic for dateFormatting & Local Testing

- dateFormatting now looks at all blueprints
- const userId updated to account for blank userId that occurs during local testing

colinfrederickson committed:
Updates Logic for dateFormatting & Local Testing

- dateFormatting now looks at all blueprints
- const userId updated to account for blank userId that occurs during local testing

colinfrederickson committed:
Update index.ts

colinfrederickson committed:
Update dateFormats.js

- accounts for additional date formats that Austin called out during testing

colinfrederickson committed:
Update dateFormats.js

- accounts for additional date formats that Austin called out during testing

johnmosesman commented on common/dateFormats.js:
```diff
@@ -0,0 +1,50 @@
+export const momentFormats = [
```
Can you convert this .js to .ts?

colinfrederickson replied:
Resolved in [dateFormats.js > dateFormats.ts](https://github.com/FlatFilers/hcm-show-config/pull/57/commits/d698ceff45aa6251def9b0d226a51d3a8e510bda)

colinfrederickson committed:
dateFormats.js > dateFormats.ts

- Addresses @johnmosesman comment

colinfrederickson committed:
dateFormats.js > dateFormats.ts

- Addresses @johnmosesman comment

colinfrederickson committed:
Addressing Merge Conflicts

colinfrederickson committed:
Merge branch 'main' into adding-back-date-formatting-hooks

colinfrederickson committed:
Merge branch 'adding-back-date-formatting-hooks' of https://github.com/FlatFilers/hcm-show-config into adding-back-date-formatting-hooks

colinfrederickson committed:
moves all .js files to .ts

- moves all .js files to .ts based on @johnmosesman request
- for all workflows using benefitsBlueprint, sheets updated to include "as any" to avoid error (handled similarly in the project workflow)

johnmosesman committed:
Remove “primary” from required type as it appears to no longer exist.

johnmosesman committed:
Convert project sheet too

colinfrederickson committed:
Update common.ts

- update necessary based on switch to .ts. importing vlookup is no longer broken.
